corehq/apps/hqadmin/management/commands/delete_related_cases.py

```diff
@@ -15,9 +15,7 @@ def add_arguments(self, parser):
         parser.add_argument('case_id', type=unicode)
         parser.add_argument('--filename', dest='filename', default='case-delete-info.csv')
 
-    def handle(self, *args, **options):
-        domain = options['domain']
-        case_id = options['case_id']
+    def handle(self, domain, case_id, **options):
```

Comment:
Is this moving options as arguments? If yes, would it be nice to keep those still in options to avoid changes in implementation?